docs/guides/avoid_blocking.md

```diff
@@ -0,0 +1,49 @@
+---
+id: avoid-blocking-fingerprints
+title: Avoid getting blocked
+---
+TODO: this is only for browser-based scraping. We should reflect in the name somehow... IMHO, browser fingerprinting does not remember that it reduces blocking.
+
+
+Your scraper might get blocked for numerous reasons. Let's narrow it down to two main reasons. The first one is a bad or blocked IP address. This topic is covered in the [proxy management guide](proxy_management.MD). The second reason we will explore more is browser fingerprints or signatures.
+Browser fingerprint is a collection of browser attributes and significant features that can show if your browser is a bot or a real user TODO: We could link the academy here in the future. Moreover, even most browsers have these unique features that allow the website to track you even within different IP addresses. This is the main reason why you should change browser fingerprints while doing browser-based scraping. In addition, it should reduce blocking significantly.
+
+Changing browser fingerprints can be a tedious job. Luckily, Apify SDK provides this feature out of the box with zero configuration necessary. Let's take a look at how it is done.
+
+ Changing browser fingerprints is available in `PuppeteerCrawler` and `PlaywrightCrawler`. You have to pass the `useFingerprints` option to the `browserPoolOptions`.
```

Comment:
formatting and indenting is messed up in this file, also lets use the same code style in examples as in the rest of the docs and codebase:
- spaces inside object literals `{ ... }`
- space after `:` (e.g. `browserPoolOptions: {`)
- trailing commas
- single quotes